src/components/PageMetaData/PageMetaData.tsx

```diff
@@ -1,17 +1,19 @@
 import { Helmet } from 'react-helmet'
 
+import { useEnvironment } from '../../hooks/useEnvironment'
+
 interface PageMetaDataProps {
-  title: string
-  description: string
-  noIndex: boolean
+  title?: string
+  description?: string
 }
 
-export function PageMetaData({ title: pageTitle, description, noIndex }: PageMetaDataProps) {
+export function PageMetaData({ title = 'Swapr', description }: PageMetaDataProps) {
+  const { isProduction } = useEnvironment()
   return (
     <Helmet>
-      <meta name="title" content={pageTitle} />
-      <meta name="description" content={description} />
-      {noIndex && <meta name="robots" content="noindex,nofollow" />}
+      <title>{title}</title>
+      {description && <meta name="description" content={description} />}
+      {!isProduction && <meta name="robots" content="noindex,nofollow" />}
     </Helmet>
```

Comment:
We discussed crawlers and SEO, do crawlers consider injected meta tags?

Reply:
No. We need SSR for that :(

Reply:
We may need to switch Next.js, and generate the main pages statically. SSR is not available on IPFS.